### PR TITLE
Create pharo11-released.md

### PR DESCRIPTION
**Pull request by tesonep**

### CODE REVIEW

File: site/news/pharo11-released.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+layout: blogpost
+publishDate: "2023-05-10"
+
+Dear Pharo users and dynamic language lovers: 
+
+We have released [Pharo](https://pharo.org/) version 11!
+
+## What is Pharo?
+
+Pharo is a pure object-oriented programming language and a powerful environment, focused on simplicity and immediate feedback.
```
Comment (pavel-krivanek) [inline site/news/pharo11-released.md:11]: Grammarly complaints:
powerful environment focused (no comma)

File: site/news/pharo11-released.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+![Screenshot of Pharo 11](https://files.pharo.org/web-images/news/Pharo11.png)
+
+- Simple & powerful language: No constructors, no types declaration, no interfaces, no primitive types. Yet a powerful and elegant language with a full syntax fitting in one postcard! Pharo is objects and messages all the way down.
+
+- Live, immersive environment: Immediate feedback at any moment of your development: Developing, testing, debugging. Even in production environments, you will never be stuck in compiling and deploying steps again!
+
+- Amazing debugging experience: Pharo environment includes a debugger, unlike anything you've seen before. It allows you to step through code, restart the execution of methods, create methods on the fly, and much more!
```
Comment (pavel-krivanek) [inline site/news/pharo11-released.md:19]: debugger unlike (no comma)

File: site/news/pharo11-released.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+
+- Pharo is yours: Pharo is made by an incredible community, with more than 100 contributors for the last revision of the platform and hundreds of people contributing constantly with frameworks and libraries.
```
Comment (pavel-krivanek) [inline site/news/pharo11-released.md:21]: people constantly contributing (swap words)

File: site/news/pharo11-released.md (hunk 1 above)
Comment (tesonep) [reply site/news/pharo11-released.md]: Thanks

File: site/news/pharo11-released.md (hunk 2 above)
Comment (tesonep) [reply site/news/pharo11-released.md]: Thanks

File: site/news/pharo11-released.md (hunk 3 above)
Comment (tesonep) [reply site/news/pharo11-released.md]: Thanks

**Pull request merged**